services/core-web/src/components/Forms/permits/GeneratePermitForm.js

```diff
@@ -1,292 +1,345 @@
-          <Col xs={24} md={12}>
-            <Field
-              id="mine_no"
-              name="mine_no"
-              label="Mine Number"
-              component={renderConfig.FIELD}
-              validate={[required]}
-              disabled
-            />
-          </Col>
-          <Col xs={24} md={12}>
-            <Field
-              id="permit_number"
-              name="permit_number"
-              label="Permit Number"
-              component={renderConfig.FIELD}
-              validate={[required]}
-              disabled
-            />
-          </Col>
-        </Row>
-        <Row gutter={32}>
-          <Col xs={24} md={12}>
-            <Field
-              id="now_number"
-              name="now_number"
-              label="Application Number"
-              component={renderConfig.FIELD}
-              validate={[required]}
-              disabled
-            />
-          </Col>
-          <Col xs={24} md={12}>
-            <Field
-              id="now_tracking_number"
-              name="now_tracking_number"
-              label="Application Tracking Number"
-              component={renderConfig.FIELD}
-              validate={[required]}
-              disabled
-            />
-          </Col>
-        </Row>
-        <Row gutter={32}>
-          <Col xs={24} md={12}>
-            <Field
-              id="permittee"
-              name="permittee"
-              label="Permittee"
-              component={renderConfig.FIELD}
-              validate={[required]}
-              disabled
-            />
-          </Col>
-          <Col xs={24} md={12}>
-            <Field
-              id="permittee_mailing_address"
-              name="permittee_mailing_address"
-              label="Permittee Mailing address"
-              component={renderConfig.AUTO_SIZE_FIELD}
-              validate={[required]}
-              disabled
-            />
-          </Col>
-        </Row>
-        <Row gutter={32}>
-          <Col xs={24} md={12}>
-            <Field
-              id="permittee_email"
-              name="permittee_email"
-              label="Permittee Email"
-              component={renderConfig.FIELD}
-              disabled
-            />
-          </Col>
-          <Col xs={24} md={12}>
-            <Field
-              id="mine_location"
-              name="mine_location"
-              label="Mine Location"
-              component={renderConfig.FIELD}
-              validate={[required]}
-              disabled
-            />
-          </Col>
-        </Row>
-        <Row gutter={32}>
-          <Col xs={24} md={12}>
-            <Field
-              id="issue_date"
-              name="issue_date"
-              label={props.isAmendment ? "Amendment Issue Date" : "Issue Date"}
-              component={renderConfig.DATE}
-              validate={[required]}
-              disabled
-            />
-          </Col>
+export const GeneratePermitForm = (props) => {
+  return (
```

Comment:
Can we revert this component back to an implicit return?

Reply:
it returns implicitly now. :)